ext/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/common/CMakeLists.txt)
  
 # --------------------------- Tests ---------------------------
 
-if (FL_BUILD_TESTS)
+# The SeqBuilder test needs some contribs modules
+if (FL_BUILD_TESTS AND FL_BUILD_CONTRIB)
```

Comment:
Can we remove this and instead [gate the `SequentialBuilder` test](https://github.com/facebookresearch/flashlight/blob/master/ext/test/CMakeLists.txt#L6) to only build if `FL_BUILD_CONTRIB` is on? You can send a separate PR for this.

Reply:
done